scripts/integration-test-all.sh

```diff
@@ -108,7 +108,7 @@ init_func() {
 
 start_func() {
     echo "starting ethermint node $i in background ..."
-    "$PWD"/build/ethermintd start --pruning=nothing --rpc.unsafe --log_level "main:info,state:info,mempool:info" \
+    "$PWD"/build/ethermintd start --pruning=nothing --rpc.unsafe \
```

Comment:
are all the log_levels defaulted to `info` or is this logging unnecessary for the integration tests?

Reply:
yeah need to fix this, good catch

Reply:
just checked and the default is ok